Update edge-proxy (1.4.0) and pe-terminanal (1.2.0)

JanneKiiskila opened the pull request:
Update edge-proxy and pe-terminal (aka edge-terminal) to the latest versions.

This requires then changes to `Dockerfile`:

Version 27.03 of `docker` does not seem to take the line 3 and 4 in anymore as previously, it ignores them. This made the build fail

```
 2 warnings found (use --debug to expand):
 - UndefinedVar: Usage of undefined variable '$http_proxy' (line 19)
 - UndefinedVar: Usage of undefined variable '$https_proxy' (line 20)
```

We also had one mixed capital usage with `FROM` and `as` -> `AS` needs to be in capital as well.

Install `golang 1.20` as it's needed with updated `pe-terminal` and `edge-proxy` - they are golang 1.20 based instead of 1.18.

JanneKiiskila committed:
Update edge-proxy (1.4.0) and pe-terminanal (1.2.0)

This requires then changes to Dockerfile:

Version 27.03 of docker does not seem to take the line 3 and 4 in
anymore as previously, it ignores them. This made the build fail

```
 2 warnings found (use --debug to expand):
 - UndefinedVar: Usage of undefined variable '$http_proxy' (line 19)
 - UndefinedVar: Usage of undefined variable '$https_proxy' (line 20)
```

We also had one mixed capital usage with FROM and as -< AS needs to
be in capital as well.

Install golang 1.20 as it's needed with updated pe-terminal and
edge-proxy - they are golang 1.20 based instead of 1.18.

petedyerarm commented on Dockerfile:
```diff
@@ -1,7 +1,7 @@
 ARG UBUNTU=20.04
-ARG http_proxy=""
-ARG https_proxy=""
+#ARG http_proxy=""
+#ARG https_proxy=""
```
Could these be removed instead of commented out?

JanneKiiskila replied:
Done!

JanneKiiskila committed:
Dockerfile - remove #ARGs

They are not adding value, the ARG for http proxy is documented
a few lines below them anyway.

Ref: review feedback from Pete.

JanneKiiskila commented:
Rebased against master and force pushed.

petedyer11 reviewed (approved):
LGTM
